cpp/cmake_modules/FindOpenSSLAlt.cmake

```diff
@@ -22,17 +22,24 @@ endif()
 if(APPLE AND NOT OPENSSL_ROOT_DIR)
   find_program(BREW brew)
   if(BREW)
-    execute_process(COMMAND ${BREW} --prefix "openssl@1.1"
-                    OUTPUT_VARIABLE OPENSSL11_BREW_PREFIX
+    execute_process(COMMAND ${BREW} --prefix "openssl"
+                    OUTPUT_VARIABLE OPENSSL_BREW_PREFIX
                     OUTPUT_STRIP_TRAILING_WHITESPACE)
-    if(OPENSSL11_BREW_PREFIX)
-      set(OPENSSL_ROOT_DIR ${OPENSSL11_BREW_PREFIX})
+    if(OPENSSL_BREW_PREFIX)
+      set(OPENSSL_ROOT_DIR ${OPENSSL_BREW_PREFIX})
     else()
-      execute_process(COMMAND ${BREW} --prefix "openssl"
-                      OUTPUT_VARIABLE OPENSSL_BREW_PREFIX
+      execute_process(COMMAND ${BREW} --prefix "openssl@3.0"
+                      OUTPUT_VARIABLE OPENSSL3_BREW_PREFIX
                       OUTPUT_STRIP_TRAILING_WHITESPACE)
-      if(OPENSSL_BREW_PREFIX)
-        set(OPENSSL_ROOT_DIR ${OPENSSL_BREW_PREFIX})
+      if(OPENSSL11_BREW_PREFIX)
```

Comment:
Should this be `OPENSSL3_BREW_PREFIX`?

Reply:
Ah, yes. Good catch!

Ah, wait. I should have used `OPENSSL30_...` for it.